test/parser/libqasm/test_libqasm.py

```diff
@@ -78,3 +109,56 @@ def test_error(parser: Parser) -> None:
 def test_wrong_gate_argument_number_or_types(parser: Parser, error_message: str, circuit_string: str) -> None:
     with pytest.raises(IOError, match=error_message):
         parser.circuit_from_string(circuit_string)
+
+
+@pytest.mark.parametrize(
+    ("expected_output", "circuit_string"),
+    [
+        (
+            """version 3.0
+
+qubit[1] q
+""",
+            """
+            version 3.0
+
+            qubit q
+
+            """,
```

Comment:
Can we write this as `"version 3.0; qubit q"`? Notice no newline at the end.

Reply:
I added the test you mentioned, as to test both ```version 3.0; qubit q``` and ```version 3.0 \n qubit q```